web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -584,7 +587,8 @@
           doCycleIn200();
         });
       },
-      logError: settings.logError,
+      logError: function() { settings.logError.apply(this, arguments) },
```

Comment:
The commit message I wrote was:

```
We were tying settings.logError directly to lift.logError, but
this didn’t allow for the options passed to init to override the
implementation because lift.logError is bound before the init
function runs. We now delegate to settings.logError using
apply, ensuring we’re always seeing the latest version of it.
```

So I think the issue is that without that, sending in new settings with an updated `logError` won't work.